src/vosk_api.h

```diff
@@ -38,6 +38,7 @@ typedef struct VoskSpkModel VoskSpkModel;
  *  which represent decoded information - words, confidences, times, n-best lists,
  *  speaker information and so on */
 typedef struct VoskRecognizer VoskRecognizer;
+typedef struct VoskGpu VoskGpu;
```

Comment:
Not needed anymore right?

Reply:
I don't know. It was added on May 2, 2020. And still lives in the GPU branch. So I have no idea if it's something intended to be used in the future, or just a dead code that is not required anymore. But if you say it's redundant, I'll remove it of course.

Reply:
Removed.